db/reporting_structure.sql

```diff
@@ -22,50 +22,50 @@ SET row_security = off;
 
 CREATE FUNCTION public.monthly_reports_insert_trigger() RETURNS trigger
```

Comment:
There's no migrations in this branch so I don't think the structure files should be included. Suggest:
```
git checkout release-92 db/reporting_structure.sql db/structure.sql db/health_structure.sql db/warehouse_structure.sql
```